releases/unreleased/changelog-location-updated-between-versions.yml

```diff
@@ -0,0 +1,20 @@
+---
+title: Changelogs location updated between versions
+category: changed
+author: Jose Javier Merchante <jjmerchante@bitergia.com>
+issue: null
+notes: |
+  The following changes have been accomplished:
+  - `notes` command moves changelog entries included in the release
+  notes are moved to a new directory named `processed`. If you are
+  running multiple release candidates, and you don't want to include
+  the same notes in sucessive release candidates, use the flag
+  `--pre-release` and will use only the changelogs not included in
+  previous calls to `notes`.
+  
+  - `semverup` command only checks new files not excluded previously
+  by `notes` command. It won't increase the version number if all the
+  changelogs are processed by notes, but still not published as a
+  final release.
+  
+  - `publish` only removes the changelogs processed by `notes` command,
```

Comment:
```suggestion
notes: |
  The flag `--pre-release`, on the command `notes`, generates the notes
  of a release candidate. Release notes include the changes in between
  releases - regular and candidate releases - while the release candidate
  notes only include the latest changes since the previous release.
```